stix_edh/bindings/cyber_profile.py

```diff
@@ -277,7 +279,7 @@ def buildChildren(self, child_, node, nodeName_, fromsubclass_=False):
             self.gds_validate_string(child_.text, node, 'compilationReason')
             obj_ = edh_common.NMTOKENS.factory()
             obj_.build(child_)
-            self.compilationReason = obj_
+            self.set_compilationReason(obj_)
```

Comment:
Is this related to sharing scope? This change makes me nervous; not sure what it impacts.